src/robot/client.ts

```diff
@@ -407,6 +414,10 @@ export class RobotClient extends EventDispatcher implements Robot {
         },
       };
 
+      if (priority && opts.webrtcOptions) {
```

Comment:
also what if priority is 0? that should still be passed through but would evaluate to false here